testsuite/gateway/envoy/config.py

```diff
@@ -140,3 +140,24 @@ def remove_all_backends(self):
             "virtual_hosts"
         ][0]["routes"] = {}
         self["envoy.yaml"] = yaml.dump(config)
+
+    @modify
+    def add_routes_for_jwt(self, backend: Backend, path_separated_prefix: str):
```

Comment:
We want to have reusable methods, this looks like an extracted copy of what you had. It needs to be possible to use this same method on a test with wristband envoy and maybe others.

Have it add config components based on the parameters passed to the method. We do that a lot in other constructing classes